odoo/addons/base/transpiler/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import sourcemap_generator
+from . import transpiler_js
```

Comment:
Having a JS transpiler in module `base` looks like an odd place to me.
Shouldn't it be in module `web`?
I don't know how difficult it would be to move it there...

Reply:
It's strange to create dependency from base to web.